omero/features/share/index.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: omero-features
+title: OMERO Features
+feature: Share
+usergroup: omero
+meta_description: Learn how to share data within OMERO.
+---
+
+        <!-- begin OMERO features -->
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h2 class="header-blue">Sharing your data in OMERO</h2>
+            <hr class="medium-10">
+            <p class="subheader">
+                Share your images and collaborate with colleagues using the permissions system.
+            </p>
+            <hr class="medium-10">
+            <div class="row small-12 medium-up-3">
+
+                <div class="omero-feature-container column text-center">
+                    <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_manage.svg">
+                    <h4>Permissions</h4>
+                    <p>Allows users to share data within groups and collaborate on their data.</p>
+                    <a class="button hollow tiny" href="https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html" target="_blank">View user guide</a>
+                    <a class="button hollow tiny" href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/server-permissions.html" target="_blank">View sysadmin guide</a>
+                </div>
+                <div class="omero-feature-container column text-center">
+                    <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_annotate.svg">
+                    <h4>Duplicate and Move the data to Share</h4>
+                    <p>Share the data accross groups using the Duplicate and Move workflow.</p>
+                    <a class="button hollow tiny" href="https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#shares-discontinued-feature" target="_blank">View user guide</a>
```

Comment:
Agreeing with @will-moore, it's unforutunate that https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#command-line-duplicating-objects doesn't show up in the left-hand panel. Perhaps having a top-level that's not CLI-specific would be good for the future.

Reply:
``Perhaps having a top-level that's not CLI-specific would be good for the future.``

Sorry, not sure I understand this particular sentence.

Reply:
> https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#command-line-duplicating-objects doesn't show up in the left-hand panel. 

afaik, doable easily, but a link-breaker, and, unfortunately, on a page which we linked already (for example from release announcement) - maybe there could be a solution, but atm do not see which...